tests/Microsoft.VisualStudio.ProjectSystem.Managed.UnitTests/Mocks/ITemporaryPropertyStorageFactory.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using System.Collections.Generic;
+using Microsoft.VisualStudio.ProjectSystem.Properties;
+using Moq;
+
+namespace Microsoft.VisualStudio.ProjectSystem
+{
+    internal static class ITemporaryPropertyStorageFactory
+    {
+        public static ITemporaryPropertyStorage Create(Dictionary<string, string>? values = null)
+        {
+            if (values is null)
+            {
+                values = new();
+            }
```

Comment:
Could use null coalescing assignment:

```suggestion
            values ??= new();
```

Reply:
I always forget about that operator!